[WPE] Unskip fast/forms layout tests

fujii opened the pull request:
#### f79a3978340b447e4cbf19f34afaaabc4d79aeb3
<pre>
[WPE] Unskip fast/forms layout tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=296872">https://bugs.webkit.org/show_bug.cgi?id=296872</a>

Unreviewed test gardening for WPE and GTK. Unskipped fast/forms layout
tests for WPE. Moved common expectations from gtk to glib.

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/gtk/fast/forms/auto-fill-button/input-strong-password-auto-fill-button-expected.txt:
* LayoutTests/platform/gtk/fast/forms/input-appearance-spinbutton-expected.txt: Added.
* LayoutTests/platform/gtk/fast/forms/input-appearance-spinbutton-up-expected.txt: Added.
* LayoutTests/platform/gtk/fast/forms/select-size-expected.png: Removed.
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/platform/wpe/fast/forms/...: Added.

Canonical link: <a href="https://commits.webkit.org/298186@main">https://commits.webkit.org/298186@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d7947bb2aa91c1f558d9496a7dfcd3b74f486d81

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/114572 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/34318 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/24781 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/120736 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/65298 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/34947 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/42878 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/87082 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/41991 "") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/117520 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/27849 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/102888 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/67473 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27035 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/21015 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/64418 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/97227 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/21128 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/123943 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/41585 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/31045 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/95902 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/41962 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/99078 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/95685 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/40849 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/18684 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/37663 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18348 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/41464 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/41050 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/44357 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/42800 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/7a61cb776934c04b034048ce69971dd4db35de50. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48885

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/d7947bb2aa91c1f558d9496a7dfcd3b74f486d81. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48885

fujii committed:
[WPE] Unskip fast/forms layout tests
https://bugs.webkit.org/show_bug.cgi?id=296872

Unreviewed test gardening for WPE and GTK. Unskipped fast/forms layout
tests for WPE. Moved common expectations from gtk to glib.

* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/gtk/fast/forms/auto-fill-button/input-strong-password-auto-fill-button-expected.txt:
* LayoutTests/platform/gtk/fast/forms/input-appearance-spinbutton-expected.txt: Added.
* LayoutTests/platform/gtk/fast/forms/input-appearance-spinbutton-up-expected.txt: Added.
* LayoutTests/platform/gtk/fast/forms/select-size-expected.png: Removed.
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/platform/wpe/fast/forms/...: Added.

Canonical link: https://commits.webkit.org/298186@main
